ddtrace/opentracer/tracer.py

```diff
@@ -330,3 +332,25 @@ def extract(self, format, carrier):  # noqa: A002
         dd_span_ctx = ot_span_ctx._dd_context
         self._dd_tracer.context_provider.activate(dd_span_ctx)
         return ot_span_ctx
+
+    def get_correlation_ids(self):
+        # type: () -> Tuple[str, str]
+        """Retrieves the Correlation Identifiers for the current active ``Trace``.
+        This helper method can be achieved manually and should be considered
+        only a shortcut.
+
+        OpenTracing users can still extract these values using the ``ScopeManager``
+        API, though this shortcut is a simple one-liner. The usage is:
+
+                trace_id, span_id = tracer.get_correlation_ids()
+
+        :returns: a tuple of strings containing the trace_id and span_id. Defaults to "0" if not found.
+        """
+        # If tracer is disabled, skip
+        if not self._enabled:
+            return "0", "0"
+
+        span = self._dd_tracer.current_span()
```

Comment:
should this just be `return self._dd_tracer.get_correlation_ids()` ?

Reply:
haha nice catch, didn't realize that at the time 😝